docs/reference/cli.md

```diff
@@ -281,3 +281,13 @@ quickwit delete --index-uri s3://quickwit-indexes/catalog
 ```bash
 quickwit delete --index-uri s3://quickwit-indexes/catalog --dry-run
 ```
+
+## Environment Variables
+
+### QUICKWIT_ENV
```

Comment:
I would point out that this is currently used only for testing purpose.

Reply:
Good call.